hydrus/metadata/drone_doc_gen.py

```diff
@@ -14,15 +14,15 @@ def drone_doc(API, BASE_URL):
                        BASE_URL)
 
     # Status Class
-    status = HydraClass("Status", "Status", "Class for drone status objects")
+    status = HydraClass("http://hydrus.com/Status", "Status", "Class for drone status objects")
```

Comment:
The way hydra-py finds resources is say we have a SCHEMA `http://schema.org` then `SCHEMA. addAction` will look for schema.org/addAction . I don't see any problem using a placeholder